### PR TITLE
Feature/issue #55

### PR DESCRIPTION
**Pull request by amba178**
Commented out has_one :resume for now in the JS model.

### CODE REVIEW

File: app/models/job_seeker.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class JobSeeker < ActiveRecord::Base
+  belongs_to :job_seeker_status
+  #has_one    :resume
```
Comment (joaopapereira) [inline app/models/job_seeker.rb]: We are missing the act_as :user.
I just added the Pull Request that enables this. Please merge the changes and add the act_as

File: app/models/job_seeker.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class JobSeeker < ActiveRecord::Base
+  belongs_to :job_seeker_status
+  #has_one    :resume 
+  has_one	   :address, as: :location
+  has_and_belongs_to_many :agency_people, :join_table => "agencies_seekers"
```
Comment (joaopapereira) [inline app/models/job_seeker.rb]: Is this the relationship of JS with JD and CM?

File: app/models/job_seeker.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  #has_one    :resume 
+  has_one	   :address, as: :location
+  has_and_belongs_to_many :agency_people, :join_table => "agencies_seekers" 
+  validates_presence_of :year_of_birth, :resume
+  YEARS = 100 
+  validate :validate_year_of_birth  
+
+
+  private 
+  	def validate_year_of_birth
```
Comment (joaopapereira) [inline app/models/job_seeker.rb]: Do you think that we will need this validation elsewhere? or just in this class?

**Comment by joaopapereira** [comment]
Do we have the job_seeker_status PR?

File: app/models/job_seeker.rb (hunk 3 above)
Comment (amba178) [inline app/models/job_seeker.rb]: Good point, just need here.  I was trying to remind myself that this is not an instance method. I might be wrong, Joao.

**Comment by amba178** [comment]
will create new one.

**Pull request closed without merging**